packages/components/src/Enumeration/Enumeration.component.js

```diff
@@ -73,7 +73,29 @@ Enumeration.propTypes = {
 	...ItemEditPropTypes,
 };
 
-function ItemsEnumeration({ items, itemsProp, searchCriteria, currentEdit }) {
+function hintClasses() {
+	return classNames({
+		[theme['tc-enumeration-hint']]: true,
+		'tc-enumeration-hint': true,
+	});
+}
+
+function EmptyListPlaceholder({ displayMode }) {
+	const EMPTY_LIST_PLACEHOLDER_DEFAULT = 'The list is empty';
```

Comment:
those two const could be moved outside the function declaration, so to avoid allocation and garbage collection for each executions